pkg/dynamicinstrumentation/testutil/sample/stacktraces.go

```diff
@@ -48,8 +48,8 @@ func inline_me_3() {
 
 //nolint:all
 //go:noinline
-func not_inlined() string {
-	return fmt.Sprintf("hello %d!", 42)
+func not_inlined() {
+	// return fmt.Sprintf("hello %d!", 42)
```

Comment:
Is this intentional? If so, I'd remove the code rather than having it commented out.

Reply:
Not intentional, fixing